draft-ietf-privacypass-architecture.md

```diff
@@ -683,76 +717,47 @@ must balance this against the reduction in Client privacy. In general,
 bounding the metadata permitted ensures that it cannot uniquely identify
```

Comment:
I wonder if we should call out here or in the next section that many token types won't have any metadata, and that we have an implicit recommendation to avoid metadata unless necessary and justified and properly analyzed...